recipes/atol-bpa-datamapper/meta.yaml

```diff
@@ -0,0 +1,36 @@
+---
+{% set version = "0.1.1" %}
+
+package:
+  name: atol-bpa-datamapper
+  version: {{ version }}
+
+source:
+  url: https://github.com/TomHarrop/atol-bpa-datamapper/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 566cf736f6f6009f92076af7d04502850350327520b4565af4b499e7f4395779
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install --ignore-installed .
+  run_exports:
+    - {{ pin_subpackage("atol-bpa-datamapper", max_pin="x.x") }}
+
+requirements:
+  host:
+    - python >=3.12,<3.13
+  run:
+    - ckanapi >=4.8
```

Comment:
python is needed in the run dependencies ... I'm a bit confused why this is working at all.